kafka-connect/kafka-connect-runtime/NOTICE

```diff
@@ -5,111 +5,21 @@ Copyright 2017-2024 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
---------------------------------------------------------------------------------
```

Comment:
I do think we need to keep this one. This is not the result of a dependency, but because we copied code from the Kite project. This comes from the parent `NOTICE`:  https://github.com/apache/iceberg/blob/main/NOTICE

Reply:
Agree, it should not be there imho.